applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -11,14 +11,17 @@ def Factory(settings, Model):
 
 class ComputeDragProcess(python_process.PythonProcess):
     def __init__(self, Model, settings ):
+        """ Auxiliary class to output total flow forces over obstacles in fluid dynamics problems.
+        """
 
         default_settings = KratosMultiphysics.Parameters("""
             {
                 "mesh_id"                   : 0,
                 "model_part_name"           : "please_specify_model_part_name",
                 "interval"                  : [0.0, 1e30],
                 "write_drag_output_file"    : true,
-                "print_drag_to_screen"      : false
+                "print_drag_to_screen"      : false,
+                "print_format"              : ""
```

Comment:
There is no possible default value for the format?

Reply:
`""` just lets python choose whatever it thinks works best as a default (I think for doubles it is "12.5g" or something of the sort)